<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1655</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
   background:transparent;
}

QLabel
{
background:transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1641</width>
      <height>891</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
background:#f0eef3 ;
border-radius:5px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
background: #cfccd4 ;
border-style: dotted;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1460</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background: #5dc8da;
color:#ffffff;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}</string>
      </property>
      <property name="text">
       <string>Daha Fazla</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway SemiBold</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color:#000000;
}</string>
      </property>
      <property name="text">
       <string>ANAODA |</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>1</y>
        <width>1041</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color:#000000;
}</string>
      </property>
      <property name="text">
       <string>MSAGE sunucularına bağlı herkesin konuşabildiği ortak bir alan.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel 
{
color:#000000;
}</string>
      </property>
      <property name="text">
       <string>MSAGE </string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>1610</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background:transparent;
color:#000000;
border-style: dotted;
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>1590</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background:transparent;
color:#000000;
border-style: dotted;
}</string>
     </property>
     <property name="text">
      <string>❏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>1570</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background:transparent;
color:#000000;
border-style: dotted;
}</string>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>1641</width>
       <height>841</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>1401</width>
         <height>791</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox
{
background:#f0eef3 ;
border-radius:15px;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1111</width>
          <height>791</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget
{
background:#e6e3ec  ;
padding:5px;

border-radius:10px;
color:#000000;
}</string>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>false</bool>
        </property>
        <property name="batchSize">
         <number>200</number>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_3">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>0</y>
          <width>281</width>
          <height>791</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget
{
background:#e6e3ec  ;
border-radius:10px;
color:#000000;
padding:5px;

}</string>
        </property>
       </widget>
      </widget>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>1420</x>
         <y>0</y>
         <width>211</width>
         <height>831</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
background: #e6e3ec;
color:#000000;
border-radius:10px;
padding:5px;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>800</y>
         <width>1361</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background:#cfccd4 ;
color:#000000;
padding:5px;
border-style: dotted;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>800</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:#5dc8da;
padding: 2px;
border-radius: 5px;
color:#ffffff;
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>1470</x>
         <y>0</y>
         <width>161</width>
         <height>831</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox
{
background: #e6e3ec;
color:#000000;
border-radius:10px;
padding:5px;
}
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: #5dc8da;
color:#ffffff;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>PROFİLİM</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: #5dc8da;
color:#ffffff;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>ODALAR </string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: #5dc8da;
color:#ffffff;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>HAKKIMIZDA</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: #5dc8da;
color:#ffffff;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>GÜVENLİK</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background: #5dc8da;
color:#ffffff;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>GİZLİLİK</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
